test/SlowTests/Issues/RavenDB_2495.cs

```diff
@@ -1,24 +0,0 @@
-// -----------------------------------------------------------------------
-//  <copyright file="RavenDB_2495.cs" company="Hibernating Rhinos LTD">
-//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
-//  </copyright>
-// -----------------------------------------------------------------------
-
-using FastTests;
-using FastTests.Client.MoreLikeThis;
-using Xunit;
-
-namespace SlowTests.Issues
-{
-    public class RavenDB_2495 : RavenTestBase
-    {
-        [Fact]
-        public void IncludesShouldWorkWithMoreLikeThis()
-        {
-            using (var x = new MoreLikeThisTests())
-            {
-                x.IncludesShouldWorkWithMoreLikeThis();
```

Comment:
Restore candidate: no idea why it was done like here, but we need to add [Fact] to `IncludesShouldWorkWithMoreLikeThis` , `TransformersShouldWorkWithMoreLikeThis1` and `TransformersShouldWorkWithMoreLikeThis2` in `MoreLikeThisTests.cs`